src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface Figure {
```

Comment:
Figure should be abstract class that hold common property for all figures (color).
It should implement 2 interfaces (AreaCalculator and Drawable or something like that)